src/elm/Pages/Secrets/Form.elm

```diff
@@ -219,6 +224,44 @@ viewImagesInput secret imageInput =
         ]
 
 
+{-| radio : takes current value, field id, title for label, and click action and renders an input radio.
+-}
+radio : String -> String -> Field -> msg -> Html msg
+radio value field title msg =
+    div [ class "form-control", Util.testAttribute <| "secret-radio-" ++ field ]
+        [ input
+            [ type_ "radio"
+            , id <| "radio-" ++ field
+            , checked (value == field)
+            , onClick msg
+            ]
+            []
+        , label [ class "form-label", for <| "radio-" ++ field ] [ strong [] [ text title ] ]
+        ]
+
+
+{-| allowCommandCheckbox : renders checkbox inputs for selecting allowcommand
+-}
+viewAllowCommandCheckbox : SecretForm -> Html Msg
+viewAllowCommandCheckbox secretUpdate =
+    section [ class "type", Util.testAttribute "" ]
+        [ h4 [ class "field-header" ]
+            [ text "Allow Commands"
+            , span [ class "field-description" ]
+                [ text "( "
+                , em [] [ text "\"No\" will disable this secret in " ]
+                , code [] [ text "commands" ]
+                , text " )"
+                ]
+            ]
+        , div
+            [ class "form-controls", class "-row" ]
```

Comment:
I understand the appeal of using `-row` here. However, I don't feel we adequately account for this from a styling perspective yet. The spacing between the radio selection controls and the labels are not distributed well to create sensible groupings in a `-row` scenario.

I would propose using `-stack` for now, which also keeps it consistent with the radio control layout on the repo settings page for controlling visibility (i know it's not a Yes/No scenario and we have a bit more content for the options).